Detectors/TPC/qc/src/Clusters.cxx

```diff
@@ -38,11 +38,27 @@ bool Clusters::processCluster(const T& cluster, const o2::tpc::Sector sector, co
     LOGP(warning, "calling denormalize() before filling");
   }
 
+  const auto& mapper = Mapper::instance();
+
   const int nROC = row < 63 ? int(sector) : int(sector) + 36;
   const int rocRow = row < 63 ? row : row - 63;
 
   const float pad = cluster.getPad();
 
+  size_t position = mapper.getPadNumber(mNClusters.getCalArray(nROC).getPadSubset(), mNClusters.getCalArray(nROC).getPadSubsetNumber(), rocRow, pad);
+
+  if ((nROC < 36)) { // IROC
+    if (position > 5279) {
```

Comment:
Do we really to hardcode this numbers? Are they not some constants of some TPC class? Same for the lines below.

Reply:
they are indeed already defined in Detectors/TPC/base/include/TPCBase/Mapper.h.
I will update that they are taken from there